my_solution.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'my_list'
+list = MyList.new(1, 2, 3, 4, 5)
+puts(list.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+puts(list.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+list.filter(&:even?)
```

Comment:
- [ ] [ OPTIONAL] Kindly I suggest you also add the check for the filter method ` puts(list.filter {|e| e.even?})` and see the  result :+1:  the
one you put is not working as expected please fix that :+1: